mwdb/web/src/components/RichAttribute/MarkedMustache.tsx

```diff
@@ -24,6 +26,10 @@ function appendToLastElement(array: string[], value: string) {
     return [...array.slice(0, -1), array[array.length - 1] + value];
 }
 
+function isFunction(obj: Object): boolean {
+    return typeof obj === "function";
+}
+
```

Comment:
I think we can use lodash `_.isFunction` for that, definition is the same :smile: https://github.com/lodash/lodash/blob/a67a085cc0612f5b83d78024e507427dab25ca2c/src/isFunction.ts#L28

Meanwhile I found that lodash is not really maintained anymore :thinking: